adafruit_pcf8574.py

```diff
@@ -177,14 +176,14 @@ def direction(self, val):
             raise ValueError("Expected INPUT or OUTPUT direction!")
 
     @property
-    def pull(self):
+    def pull(self) -> digitalio.Pull:
```

Comment:
Looks like this only returns `digitalio.Pull.UP`, so we should return that as a `Literal`.